### ISSUE TITLE
The category UIViewController+MFSideMenuAdditions is not included in the pod dependency

### ISSUE THREAD

**Issue by vtsaplin** [mikefrederick/MFSideMenu#113]

**Comment by chourobin** [comment]
Hi, I am unable to reproduce this issue. What version of cocoapods are you using and do u have a sample project?

**Comment by vtsaplin** [comment]
Hi Robin,

Here is what the pod script is saying: "Using MFSideMenu (0.5.3)"

In MFSideMenu folder I have 3 files:
MFSideMenu.h, MFSideMenuContainerViewController.h and
MFSideMenuContainerViewController.m

The content of MFSideMenu.h is:
*
*
_//_

_//  MFSideMenu.h_

_//  MFSideMenuDemoBasic_

_//_

_//  Created by Michael Frederick on 5/5/13._

_//  Copyright (c) 2013 Frederick Development. All rights reserved._

_//_

*
*
_#import "MFSideMenuContainerViewController.h"_

which is different from what I can see on github.

Vitaly

2013/8/23 Robin Chou notifications@github.com

**Comment by chourobin** [comment]
Looks like you are right, the category file is not included in the 0.5.3 tag release.  The temporary workaround is to use the master branch:

```
pod 'MFSideMenu', github: "https://github.com/mikefrederick/MFSideMenu"
```

**Comment by mikefrederick** [comment]
Any idea why this is? I haven't looked yet but the pod spec requires MFSideMenu/\* I believe.

**Comment by chourobin** [comment]
The podspec is correct, its just the git tag doesn't include the two category files.

Edit: just to clarify, I think the 0.5.3 tag doesn't include the commits that has the two UIViewController category files.

**Comment by mikefrederick** [comment]
Included in 0.5.4, thanks for pointing this out.
